SourceX/storm/storm.cpp

```diff
@@ -21,41 +21,12 @@ bool directFileAccess = false;
 
 static std::string getIniPath()
 {
-	char path[DVL_MAX_PATH];
-	GetPrefPath(path, DVL_MAX_PATH);
-	std::string result = path;
-	result.append("diablo.ini");
-	return result;
+	return GetPrefPath() + "diablo.ini";
 }
 
 static radon::File ini(getIniPath());
 static Mix_Chunk *SFileChunk;
 
-void GetBasePath(char *buffer, size_t size)
-{
-	char *path = SDL_GetBasePath();
-	if (path == NULL) {
-		SDL_Log(SDL_GetError());
-		buffer[0] = '\0';
-		return;
-	}
-
-	snprintf(buffer, size, "%s", path);
-	SDL_free(path);
-}
-
-void GetPrefPath(char *buffer, size_t size)
-{
-	char *path = SDL_GetPrefPath("diasurgical", "devilution");
-	if (path == NULL) {
-		buffer[0] = '\0';
-		return;
-	}
-
-	snprintf(buffer, size, "%s", path);
-	SDL_free(path);
-}
-
 void TranslateFileName(char *dst, int dstLen, const char *src)

```

Comment:
This looks like another SourceX function that would benifit from a change of signature

Reply:
Done